### DISCUSSION TITLE
Why Forem don't use elasticsearch anymore?

### DISCUSSION THREAD

**Discussion by AlexandreL0pes** [forem/forem#18526] [Q&A] [answered]
First of all, congratulations for the amazing project!

I'm learning about elasticsearch at the moment and looking at the project I've seen this PR with the elasticsearch removal: https://github.com/forem/forem/pull/13606

What were the reasons that led to the removal? Currently, only the postgres is performing well? Any downsides with the removal?

**Comment by maestromac** [comment, marked as answer]
For what we need Postgres is performant enough and require less resource overhead. It all depends and comes down to budget and platform scale.
